lib/util.c

```diff
@@ -0,0 +1,26 @@
+#include <stdlib.h>
+#include <errno.h>
+
+#include "util.h"
+
+long libfuse_strtol(const char *str)
```

Comment:
Is this not confusing? Forcing a signed type to be effectively unsigned?

Reply:
For me as well, the wrapper I had in my mind is more like

int libfuse_strtol(const char *str, long *res);

Return is then 0/error-code and res provides the result.

@joannekoong Sorry for my late reply, was concentrating on fuse-uring. Do want to make the update above, or shall I do that for you?

Reply:
Ah I didn't realize "strtol" also takes in negative numbered strings. I'll fix this up and resubmit